From 855c333ac0d16350f895f1d1b9391e9cbb0b6db4 Mon Sep 17 00:00:00 2001
From: Koen Kooi <koen@dominion.thruhere.net>
Date: Wed, 26 Jan 2011 13:27:20 +0100
Subject: [PATCH 2/6] Revert "Set virtual size when configuring framebuffer"

Virtual size too big, so revert it.

This reverts commit 2653ef07883fbd1a5e7025f9300cf89b79ba429a.

Upstream-Status: Pending

Signed-off-by: Martin Jansa <Martin.Jansa@gmail.com>
---
 src/omapfb-crtc.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/src/omapfb-crtc.c b/src/omapfb-crtc.c
index cbeff35..9655db2 100644
--- a/src/omapfb-crtc.c
+++ b/src/omapfb-crtc.c
@@ -98,8 +98,6 @@ OMAPFBCrtcCommitChangeMode (xf86CrtcPtr crtc)
 	v = ofb->state_info;
 	v.xres = mode->HDisplay;
 	v.yres = mode->VDisplay;
-	v.xres_virtual = crtc->scrn->virtualX;
-	v.yres_virtual = crtc->scrn->virtualY;
 	v.activate = FB_ACTIVATE_NOW;
 	v.pixclock = KHZ2PICOS(mode->Clock ? mode->Clock : 56000);
 	v.left_margin = mode->HTotal - mode->HSyncEnd;
-- 
1.8.0

